tests/network/link_state.go

```diff
@@ -0,0 +1,229 @@
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+const (
+	defaultInterfaceName = "default"
+	bridgeInterfaceName2 = "bridge2"
+	bridgeInterfaceNAD2  = "bridge2nad"
+	bridgeInterfaceName3 = "bridge3"
+	bridgeInterfaceNAD3  = "bridge3nad"
+	regexLinkStateUP     = "'state[[:space:]]+UP'"
+	regexLinkStateDOWN   = "'state[[:space:]]+DOWN'"
+	ipLinkTemplate       = "ip -one link | grep %s | grep -E %s\n"
+)
+
+var _ = SIGDescribe("interface state up/down", func() {
+
+	var (
+		vmName           types.NamespacedName
+		mac1, mac2, mac3 net.HardwareAddr
+	)
+	BeforeEach(func() {
+		var ifaceDefault *v1.Interface
+		var iface2 *v1.Interface
+
+		var err error
+		_, err = libnet.CreateNetAttachDef(context.Background(), testsuite.GetTestNamespace(nil),
+			libnet.NewBridgeNetAttachDef(bridgeInterfaceNAD2, "br02"))
+		Expect(err).NotTo(HaveOccurred())
+		_, err = libnet.CreateNetAttachDef(context.Background(), testsuite.GetTestNamespace(nil),
+			libnet.NewBridgeNetAttachDef(bridgeInterfaceNAD3, "br03"))
+		Expect(err).NotTo(HaveOccurred())
+
+		mac1, err = libnet.GenerateRandomMac()
+		Expect(err).NotTo(HaveOccurred())
+		mac2, err = libnet.GenerateRandomMac()
+		Expect(err).NotTo(HaveOccurred())
+		mac3, err = libnet.GenerateRandomMac()
+		Expect(err).NotTo(HaveOccurred())
+
+		ifaceDefault = libvmi.InterfaceWithMac(
+			pointer.P(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+			mac1.String())
+		iface2 = libvmi.InterfaceWithMac(libvmi.InterfaceWithState(
+			pointer.P(libvmi.InterfaceDeviceWithBridgeBinding(bridgeInterfaceName2)),
+			v1.InterfaceStateLinkDown),
+			mac2.String())
+
+		vmi := libvmifact.NewFedora(
+			libvmi.WithInterface(*ifaceDefault),
+			libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			libvmi.WithInterface(*iface2),
+			libvmi.WithNetwork(libvmi.MultusNetwork(bridgeInterfaceName2, bridgeInterfaceNAD2)),
+		)
+
+		vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(v1.RunStrategyAlways))
+		vm, err = kubevirt.Client().VirtualMachine(testsuite.GetTestNamespace(nil)).Create(context.Background(), vm, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		Eventually(ThisVM(vm)).WithTimeout(3 * time.Minute).WithPolling(3 * time.Second).Should(HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
+		vmi, err = kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		vmName = types.NamespacedName{Name: vmi.GetName(), Namespace: vmi.GetNamespace()}
+
+	})
+
+	Context("VM with one link up, one link down, one hot-plugged down", func() {
+
+		It("status and guest should show correct iface state", func() {
+			vmi := WaitForStatusToShowExpectedStates(vmName, map[string]v1.InterfaceState{
+				defaultInterfaceName: v1.InterfaceStateLinkUp,
+				bridgeInterfaceName2: v1.InterfaceStateLinkDown,
+			})
+			Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+			var timeout = time.Second * 5
```

Comment:
nit: Five seconds reads nicer IMO
```suggestion
			timeout := 5 * time.Second
```